src/components/shared/misc/ImageUploader.vue

```diff
@@ -263,13 +266,23 @@
                         })
                     })
                     $(this.$refs.select).val('')
+                    this.imageAdded = false
                 })
             },
         },
 
         computed: {
             cors() {
-                return this.src.length ? `${this.src}?v=cors` : ''
+                return this.getSrc.length ? `${this.getSrc}?v=cors` : ''
+            },
+
+            getSrc: {
```

Comment:
Wasn't the idea of this computed value to handle the defaultSrc prop being passed in?
Shouldn't the get method look something like this?

```js
get() {
  return this.src || this.defaultSrc
},
```
If not, this refactor doesn't actually do anything, right?